packages/next/config.d.ts

```diff
@@ -153,4 +153,11 @@ export interface PlatformaticNextJsStackable {
   application?: {
     basePath?: string;
   };
+  deploy?: {
```

Comment:
why is this called deploy?

Reply:
This object holds both `build`, `install` and `start` commands.